src/oviators/OviatorsExchangeMinterModule.sol

```diff
@@ -104,45 +86,14 @@ contract OviatorsExchangeMinterModule is
     function initializeWithData(bytes memory) external {
     }
 
-    function setColorLimits(ColorSetting[] calldata colorSettings)
```

Comment:
happy to chat through this over the phone, but we will need to keep `setColorLimits` / the concept of colors here. Even though we aren't using colors to determine the images, we still use them to keep track of the physical inventory of glasses (there's only a certain number of silver oviators, gold oviators, for example).

Reply:
ooooh my bad 😂

so we need to bring those color settings back, and also re-add it to the `exchange` function, correct?

Reply:
Yes i think we can just fully revert [f1e45d8](https://github.com/salvinoarmati/zora-drops-extensions/pull/1/commits/f1e45d8973f72968e9742d579607d32b01176767) and we should be good.